decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -33,7 +33,10 @@ def icon(name, options = {})
       html_properties["aria-label"] = options[:aria_label]
       html_properties["role"] = options[:role]
       html_properties["aria-hidden"] = options[:aria_hidden]
-      html_properties["class"] = "icon icon--#{name} #{options[:class]}"
+
+      icon_class = "icon"
+      icon_class = "" if options[:removeIconClass]
```

Comment:
Yes I know @Dor3nz, but it's weird to have a camelCase option. We can submit a PR at decidim-design to fix it.